ros2pkg/ros2pkg/api/create.py

```diff
@@ -17,8 +17,9 @@
 import sys
 
 import em
+import pkg_resources
 
-RESOURCE_PATH = os.path.join(os.path.dirname(__file__), '..', 'resource')
+RESOURCE_PATH = pkg_resources.resource_filename('ros2pkg', 'resource/')
```

Comment:
Why the trailing slash here?

Reply:
think this usage in general might be off now that I've read [this](http://setuptools.readthedocs.io/en/latest/pkg_resources.html#basic-resource-access): "Do not use os.path routines to manipulate resource paths, as they are not filesystem paths.". I was going for a minimal change but will replicate https://github.com/ament/ament_package/blob/28308125089b607f2797abee577371d7ee9a0934/ament_package/templates.py#L24 exactly

Reply:
well aren't I learning a lot today..! `resource_filename` will _return_ a filesystem path, I think that comment I referenced above is referring to the arguments being passed _in_, where we aren't using `os.path` routines anyway. Still, I added 99f0127967c799edea03d018037790f038b40170. If I understood correctly, `resource_filename` will potentially extract files/directories if the package is installed zipped, and this way it will only extract the files when they're needed, as opposed to the whole resource directory being extracted (or we can swap back).